app/views/home/_header.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-expand-xl navbar-light bg-warning">
```

Comment:
`<nav class="navbar navbar-expand-lg navbar-dark bg-dark">`

Reply:
No need to change color???I was just want to look it beautiful.